app/src/screens/dev/DevPrivateKeyScreen.tsx

```diff
@@ -0,0 +1,110 @@
+// SPDX-FileCopyrightText: 2025 Social Connect Labs, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+// NOTE: Converts to Apache-2.0 on 2029-06-11 per LICENSE.
+
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
accidentally removed this screen in this pr - https://github.com/selfxyz/self/pull/994/files#diff-c2729da81fe0f51e63433d07376e4db00e1643cfefc5c7605cc775c158bf6e99